scripts/super-linter.sh

```diff
@@ -4,13 +4,27 @@ set -euo pipefail
 
 pushd "$(dirname "$0")/.."
 
-docker pull ghcr.io/super-linter/super-linter:latest
+if command -v docker >/dev/null 2>&1; then
+	RUNTIME=docker
+	MOUNT_OPTS=rw
+elif command -v podman >/dev/null 2>&1; then
+	RUNTIME=podman
+	# Fedora, by default, runs with SELinux on. We require the "z" option for bind mounts.
+	# See: https://docs.docker.com/engine/storage/bind-mounts/#configure-the-selinux-label
+	# See: https://docs.podman.io/en/stable/markdown/podman-run.1.html section "Labeling Volume Mounts"
+	MOUNT_OPTS="rw,z"
+else
+	echo "Unable to find a suitable container runtime such as Docker or Podman. Exiting."
+	exit 1
+fi
 
-docker run --rm \
+$RUNTIME image pull ghcr.io/super-linter/super-linter:latest
```

Comment:
Post-merge we should pin this and add the appropriate renovate metadata so this gets updated automatically.